browser/components/MarkdownEditor.js

```diff
@@ -84,6 +83,10 @@ class MarkdownEditor extends React.Component {
           this.refs.preview.focus()
         }
         eventEmitter.emit('topbar:togglelockbutton', this.state.status)
+
+        const newConfig = Object.assign({}, this.props.config)
```

Comment:
Please change `this.props.config` to `config`

Reply:
Done. Thx